README.md

```diff
@@ -112,31 +83,20 @@ Initialize your repository's `.gitignore` with the Node.js template from [`githu
 
 1. `cd` into your frontend directory.
 
-1. If you already set up ESLint when you created the project, you should have a `.eslintrc.json` file already, and you can skip this step.
-
-   Otherwise, run `npx next lint` to generate `.eslintrc.json` ([docs](https://nextjs.org/docs/app/building-your-application/configuring/eslint)), and answer the prompt as follows:
-
-   <dl>
-     <dt>How would you like to configure ESLint?</dt>
-     <dd>Strict</dd>
-   </dl>
-
-   If there is still no `.eslintrc.json`, see [this workaround](https://github.com/vercel/next.js/issues/50761#issuecomment-1666057683).
-
-1. Download some config files from this repository:
+1. Install necessary dependencies
 
    ```sh
-   for file in .eslintignore .prettierignore .prettierrc.json; do curl -O https://raw.githubusercontent.com/TritonSE/linters/main/$file; done
+   npm install --save-dev eslint prettier @antfu/eslint-config
    ```
 
-1. Install the necessary packages:
+1. If you already set up ESLint when you created the project, you should have a `eslint.config.mjs`, in which case, you should replace the contents of that file with [this file](nextjs.eslint.config.mjs). If you don't have an `eslint.config.mjs` file, download [this file](nextjs.eslint.config.mjs) and rename it to `eslint.config.mjs`.
+
+1. Download some config files from this repository:
 
    ```sh
-   npm install --save-dev @typescript-eslint/parser @typescript-eslint/eslint-plugin eslint-config-next eslint-config-prettier eslint-plugin-import prettier
+   for file in .prettierignore prettier.config.mjs eslint.config.mjs; do curl -O https://raw.githubusercontent.com/TritonSE/linters/main/$file; done
```

Comment:
I think we should remove `eslint.config.mjs` from this command because that file doesn't exist in the linters repo and adding `eslint.config.mjs` is covered by the previous paragraph's instructions